pkg/kn/commands/service/export.go

```diff
@@ -323,13 +336,19 @@ func revisionListSortFunc(revisionList *servingv1.RevisionList) func(i int, j in
 }
 
 func stripIgnoredAnnotationsFromService(svc *servingv1.Service) {
-	for _, annotation := range IGNORED_SERVICE_ANNOTATIONS {
+	for _, annotation := range IgnoredServiceAnnotations {
 		delete(svc.ObjectMeta.Annotations, annotation)
 	}
 }
 
 func stripIgnoredAnnotationsFromRevision(revision *servingv1.Revision) {
```

Comment:
It seems we need to strip those labels from Revision as well, see the CI [results](https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_client/1194/pull-knative-client-integration-tests/1351453264758968320#1:build-log.txt%3A627).